src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractFile.java

```diff
@@ -68,10 +69,19 @@ public void setOriginalFileName(String originalFileName) {
         this.originalFileName = originalFileName;
     }
 
+    public String getFullyQualifiedRequestedCashAdvancesPdpFileName() {
+        return fullyQualifiedRequestedCashAdvancesPdpFileName;
+    }
+
+    public void setFullyQualifiedRequestedCashAdvancesPdpFileName(String fullyQualifiedRequestedCashAdvancesPdpFileName) {
+        this.fullyQualifiedRequestedCashAdvancesPdpFileName = fullyQualifiedRequestedCashAdvancesPdpFileName;
+    }
+
     public String toString() {
         StringBuilder sb = new StringBuilder(" batchDate: ").append(batchDate).append(" recordCount: ").append(recordCount);
         sb.append(" journalAmountTotal: ").append(journalAmountTotal).append(" batchId: ").append(batchId);
         sb.append(" originalFileName: ").append(originalFileName);
+        sb.append(" fullyQualifiedRequestedCashAdvancesPdpFileName: ").append(fullyQualifiedRequestedCashAdvancesPdpFileName);
```

Comment:
another place that could be simplied by using toStringBuilder

Reply:
Same as above.